backup/backup.go

```diff
@@ -188,6 +188,181 @@ func (c *BackupCommand) BackupInfo(cliConnection plugin.CliConnection, backupId
 
 }
 
+func (c *BackupCommand) ListBackupsByInstanceGuid(cliConnection plugin.CliConnection, serviceInstanceGuid string) {
+	fmt.Println("Getting the list of  backups in the org", AddColor(helper.GetOrgName(helper.ReadConfigJsonFile()), cyan), "/ space", AddColor(helper.GetSpaceName(helper.ReadConfigJsonFile()), cyan), "/ service instance GUID", AddColor(serviceInstanceGuid, cyan), "...")
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	client := GetHttpClient()
+
+	var userSpaceGuid string = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	var guid string = serviceInstanceGuid
+	var ServiceName string = guidTranslator.FindInstanceName(cliConnection, guid, nil)
+
+	if ServiceName == "" {
```

Comment:
Add additional check here, if the instance is deleted. Or skip this check if the instance is deleted.